chore: Cleaner wording on BTC Staking overview closing (#62)
babylonchain/babylonchain.github.io: docs/user-guides/btc-staking-testnet/deployment-overview.md

could self-delegate BTC Signet tokens via the staking web application provided
by the Babylon team.

CometBFT validators and Finality providers are independent entities - you can be

> any of the 2. For a more complete understanding of the BTC Staking protocol, we

Reviewer: ```suggestion
any of the 2. To fully participate in the testnet, we
```
